.rubocop.yml

```diff
@@ -61,7 +62,7 @@ Style/FileName:
 Style/HashSyntax:
   Severity: fatal
   Enabled: true
-  EnforcedStyle: hash_rockets
+  EnforcedStyle: ruby19_no_mixed_keys
```

Comment:
It's quite unlikely that you'll get that situation though... 99.9% percent of hashes will only have symbols on the left.